sklearn/preprocessing/data.py

```diff
@@ -1698,14 +1698,11 @@ class OneHotEncoder(BaseEstimator, TransformerMixin):
 
     Parameters
     ----------
-    n_values : 'auto', int or array of ints
-        Number of values per feature.
+    classes : 'auto', 2D array of ints or strings or both.
```

Comment:
Oh sorry, I did not realize that the parameter was being deprecated from the diff view. +1 for values or catergories